cli/cli.go

```diff
@@ -614,6 +614,26 @@ func listToxics(toxics toxiproxy.Toxics, stream string) {
 	}
 }
 
+const fallbackUrl = "http://localhost:8474"
+
+func defaultUrl() string {
+	// For our purposes an unset variable is treated the same as a variable set
+	// to an empty string
+	url, _ := os.LookupEnv("TOXIPROXY_URL")
+	if url != "" {
+		return url
+	}
+
+	host, _ := os.LookupEnv("TOXIPROXY_HOST")
+	port, _ := os.LookupEnv("TOXIPROXY_PORT")
+
+	if host != "" && port != "" {
+		return fmt.Sprintf("http://%s:%s", strings.TrimPrefix(host, "http://"), port)
```

Comment:
I would skip support single TOXIPROXY_HOST and TOXIPROXY_PORT if there is no flag for it exists. To have consitency.